### ISSUE TITLE
Defold - Cant sign into my Google account(403 - forbidden)

### ISSUE THREAD

**Issue by emrecan29** [defold/editor2-issues#1645]
### Steps to reproduce

<hr/>

<table>
<tr><td>Build time</td><td>2018-02-05T11:33:32.956312</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>5687f96313ec6f4b7bb914617b67764206ced10b</td></tr>
<tr><td>Defold engine sha</td><td>c70ad190898e790f8284e8dab0e31c9090476be9</td></tr>
<tr><td>Defold version</td><td>1.2.120</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/defold/editor2/?query=id%3A"e2d796f03a6243bb85f46a9cce89bab6"'>e2d796f03a6243bb85f46a9cce89bab6</a></td></tr>
<tr><td>Java version</td><td>1.8.0_102-b14</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Windows 10</td></tr>
<tr><td>OS version</td><td>10.0</td></tr>
<table>

**Comment by erikangelin-king** [comment]
First of all this exception should be caught and presented, but was recently accidentally moved out from it's try-catch scope.
A common reason for the exception is the windows firewall blocking the process, did a dialog pop up asking if java should be allowed to access the internet?

**Comment by emrecan29** [comment]
There was no dialog that popped up. I did get a message on my phone saying that someone logged into your account from a different device. I clicked on yes it's me but its still blocking it. When i login using defold 2 it sais succesfully logged in but when I go back to defold the error shows up. In defold one it doesnt work either. Tried disabling windows firewall. Also went to internet explorer and disabled everything that had to do with protiection still same problem.
